config/locales/en.yml

```diff
@@ -3577,6 +3577,8 @@ See the %{link} to find out more about %{sitename}'s features and to start using
     server_error: "Server error"
     shipping_method_names:
       UPS Ground: "UPS Ground"
+      pick_up: "Pick-up at the farm"
+      delivery: "Signed, sealed, delivered"
```

Comment:
I don't really understand why does the orders page call a translation on the name of the shipping method. In order to filter for shipping methods I had to create these two, and add the keys - otherwise the page would blow up with the missing translation error.

Is this really intended?

Reply:
Yes, there are two *types* of shipping methods. So we still have the name like "Cash with no coins" but then also add if it's pickup or delivery. That field is used internally to pre-fill the shipping address with the hub's address on pickup. We don't need the customer's shipping address then.